base/operators.jl

```diff
@@ -233,7 +234,7 @@ const ≡ = ===
     !==(x, y)
     ≢(x,y)
 
-Always gives the opposite answer as [`===`](@ref).
+Always gives the opposite answer as [`===`](@ref). Beware, it's not the opposite of [`==`](@ref).
```

Comment:
```suggestion
Always gives the opposite answer to [`===`](@ref). Beware, it's not the opposite of [`==`](@ref).
```
? I'm not a native speaker, but you made the corresponding change above.

Reply:
I'm neither a native speaker, so I might be wrong. The "to"/"as" was a bit of a distraction. What do you think about the rest?

Reply:
https://english.stackexchange.com/questions/136940/opposite-to-or-opposite